app/controllers/admin/poll/booths_controller.rb

```diff
@@ -2,6 +2,7 @@ class Admin::Poll::BoothsController < Admin::Poll::BaseController
   load_and_authorize_resource class: "Poll::Booth"
 
   def index
+    @booths = @booths.search params[:search].strip if params[:search]
```

Comment:
Just curious: is the `strip` necessary? :thinking:

Reply:
yeah, will remove it, just thinking about some unusual cases)